src/TimeGrid.jsx

```diff
@@ -77,9 +80,24 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
+    }
+
     //this.checkOverflow()
   }
 
+  componentWillReceiveProps(nextProps) {
+    const { start, scrollTop } = this.props;
+    // When paginating, reset scroll
+    if (!dates.eq(nextProps.start, start) || nextProps.scrollTop !== scrollTop) {
+      this.adjustScroll();
```

Comment:
yes, because the scroll needs to be updated on `componentDidUpdate`, when it's all flushed to the DOM. So I calculate a new scrollRatio property in the object and apply it once rendered.